next.config.js

```diff
@@ -9,7 +9,7 @@ const LANGUAGES = ['en', 'zh', 'ru', 'pt', 'fr', 'es', 'de'];
 
 module.exports = withTypescript({
   webpack: (config, options) => {
-    // Alias @material-ui/core peer dependency imports form the following modules to our sources.
+    // Alias @material-ui/core peer dependency imports from the following modules to our sources.
     config = withTM({
       transpileModules: ['notistack', '@material-ui/pickers', 'material-table'],
```

Comment:
Should the packages using `classnames` be listed here instead?

Reply:
Only if you want to use babel aliases instead.

Reply:
Alright, why do we transpile `'notistack', '@material-ui/pickers', 'material-table'` instead of just using the webpack alias?